docs/sdk/src/reference_docs/frame_logging.rs

```diff
@@ -114,3 +117,23 @@
 //! [`crate::reference_docs::wasm_meta_protocol`]). The runtime uses a set of host functions under
 //! [`sp_io::logging`] and [`sp_io::misc`] to emit all logs and prints. You typically do not need to
 //! use these APIs directly.
+//!
+//! ## Using Logging in Production
+//!
+//! Note that within FRAME, reading storage values only for the purpose of logging is dangerous, and
+//! can lead to consensus issues. This is because with the introduction of
```

Comment:
You should mention that you should only read something that was read 10000% before.

Or maybe it should be said that there should not be any conditional reads (like based on the log target etc)

Reply:
made it more clear.